apps/docs/content/guides/realtime/postgres-changes.mdx

```diff
@@ -1347,6 +1347,10 @@ myChannel.updateAuth("fresh-token")
 
 ## Limitations
 
+### Delete events are not filterable
+
+When tracking Delete events with Postgres Changes you are not able to apply Filters on the received records. This is due to a limitation with the way we are pulling changes from Postgres where filters can't be applied. We're working on solutions to this limitation.
```

Comment:
```suggestion
You can't filter `DELETE` events when tracking Postgres Changes. This limitation is due to the current way Realtime polls for changes from Postgres.
```

@filipecabaco you can also mention this [issue](https://github.com/supabase/walrus/issues/53) so folks can track it. @charislam would love your thoughts on doing so.

Reply:
I'm not a huge fan of linking issues in docs because it feels less polished, which can make the product seem like a work in progress. I recognize that's a very subjective feeling, though, so if you think it would really help, I'm not going to block it 😄

Reply:
I will keep the changes that @charislam suggested and lets keep the issue out for now and lets see the feedback